docs/UserGuide.md

```diff
@@ -6,8 +6,28 @@
 If you can type fast, `FridgeFriend` can track your cold or frozen groceries faster and easier than any other apps.
 It is written in Java, and has more than 3.2kLoC.
 
-* Table of Contents
-  {:toc}
+## Contents
+
+* [User Guide](#user-guide)
```

Comment:
can we remove this bullet point? then we can shift everything below one tab to the left

Reply:
Sure, implemented.